app/src/main/python/gogdl/process.py

```
@@ -0,0 +1,138 @@
+        """Iterator that yields all the children of a process"""
+        for child in self.children:
+            yield child
+            yield from child.iter_children()
+
+    def wait_for_finish(self):
+        """Waits until the process finishes
+        This only works if self.pid is a child process of Lutris
+        """
+        try:
+            pid, ret_status = os.waitpid(int(self.pid) * -1, 0)
+        except OSError as ex:
```

Ruff (0.12.2)

Local variable `ex` is assigned to but never used

Remove assignment to unused variable `ex`

(F841)